examples/common/weblog.yml

```diff
@@ -1,67 +0,0 @@
-name: weblog-common
```

Comment:
Were you not able to still use weblog-common and override in a different sample file of your own?  The purpose of this setup was to allow multiple iterations of things which override the base sample so we could create more composable configs.